### PR TITLE
Document metrics and update overview section

### PR DESCRIPTION
**Pull request by yolossn**

### CODE REVIEW

**Comment by andrewpollack** [comment]
Thank you for the initiative @yolossn 😄 

I've made some grammar suggestions to help with the overall flow, and this is a fantastic resource for people who are starting out. Apologies if I over-commented as this is my first PR review for this project, and hope this was helpful!

**Comment by yolossn** [comment]
@andrewpollack  sorry I am not able to see any suggestions. Am I missing something here?

**Review by andrewpollack — CHANGES_REQUESTED** [review]

**Comment by andrewpollack** [comment]
> @andrewpollack sorry I am not able to see any suggestions. Am I missing something here?

My bad, I think I forgot to publish! Try checking now

File: content/docs/introduction/overview.md [hunk 1]
```diff
@@ -33,6 +37,12 @@ Prometheus's main features are:
 * targets are discovered via service discovery or static configuration
 * multiple modes of graphing and dashboarding support
 
+### What are metrics ?
+
+Metrics in layman terms is a standard for measurement. What we want to measure differs from application to application. For a web server it can be request times, for a database it can be number of active connections or number of active queries etc.
```
[the remark this answers is not in the document]
Comment (mhausenblas) [reply content/docs/introduction/overview.md:42]: s/layman/layperson

File: content/docs/introduction/overview.md [hunk 2]
```diff
@@ -33,6 +37,12 @@ Prometheus's main features are:
...
+
+Metrics play an important role in understanding why your application is working in a certain way. Lets assume you are running a web application and find that the application is slow. You will need some information to find out what is happening with your application. For example the application can become slow when the number of requests are high. If you have the request count metric you can spot the reason and increase the number of servers to handle the heavy load. Whenever you are defining the metrics for your application you must put on your detective hat and ask this question what all information will be important for me to debug if any issue occurs in my application?
```
[the remark this answers is not in the document]
Comment (mhausenblas) [reply content/docs/introduction/overview.md:44]: I wonder if it would help here if we would be more explicit with the example. So, rather than talking in general about a web app and requests, how about a concrete example using HTTP API, GET requests, etc.?

**Review by mhausenblas — COMMENTED** [review]
LGTM!

File: content/docs/introduction/overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yolossn) [reply content/docs/introduction/overview.md:44]: @mhausenblas I think giving a simple yet generic example will be more apt than getting into something specific.

**Comment by hdost** [comment]
Relates to #1856

File: content/docs/introduction/overview.md [hunk 3]
```diff
@@ -33,6 +35,14 @@ Prometheus's main features are:
 * targets are discovered via service discovery or static configuration
 * multiple modes of graphing and dashboarding support
 
+### What are metrics ?
+
+In layperson terms, metrics are a standard for measurement. What we want to measure differs from application to application. For a web server it can be request times, for a database it can be number of active connections or number of active queries etc.
```
Comment (RichiH) [inline content/docs/introduction/overview.md:40]:
```suggestion
In layperson terms, metrics are numeric measurements, time series mean that changes are recorded over time. What users want to measure differs from application to application. For a web server it might be request times, for a database it might be number of active connections or number of active queries etc.
```

File: content/docs/introduction/overview.md [hunk 4]
```diff
@@ -33,6 +35,14 @@ Prometheus's main features are:
...
+
+Metrics play an important role in understanding why your application is working in a certain way. Lets assume you are running a web application and find that the application is slow. You will need some information to find out what is happening with your application. For example the application can become slow when the number of requests are high. If you have the request count metric you can spot the reason and increase the number of servers to handle the heavy load.
```
Comment (RichiH) [inline content/docs/introduction/overview.md:42]:
```suggestion
Metrics play an important role in understanding why your application is working in a certain way. Lets assume you are running a web application and find that the application is slow. You will need some information to find out what is happening with your application. For example the application can become slow when the number of requests are high. If you have the request count metric you can spot the reason and increase the number of servers to handle the load. 
```

File: content/docs/introduction/overview.md [hunk 5]
```diff
@@ -33,6 +35,14 @@ Prometheus's main features are:
 * targets are discovered via service discovery or static configuration
 * multiple modes of graphing and dashboarding support
 
+### What are metrics ?
+
+In layperson terms, metrics are a standard for measurement. What we want to measure differs from application to application. For a web server it can be request times, for a database it can be number of active connections or number of active queries etc.
+
+Metrics play an important role in understanding why your application is working in a certain way. Lets assume you are running a web application and find that the application is slow. You will need some information to find out what is happening with your application. For example the application can become slow when the number of requests are high. If you have the request count metric you can spot the reason and increase the number of servers to handle the heavy load. 
+
+Whenever you are defining the metrics for your application you must put on your detective hat and ask this question what all information will be important for me to debug if any issue occurs in my application?
```
Comment (RichiH) [inline content/docs/introduction/overview.md:44]: That feels weird in an intro section.

**Review by RichiH — CHANGES_REQUESTED** [review]

File: content/docs/introduction/overview.md [hunk 6]
```diff
@@ -33,6 +35,14 @@ Prometheus's main features are:
...
+
```
Comment (RichiH) [inline content/docs/introduction/overview.md:45]:
```suggestion
```
